controllers/servers/csi/csi_controller_server.py

```diff
@@ -99,6 +99,11 @@ def CreateVolume(self, request, context):
                         "volume was not found. creating a new volume with parameters: {0}".format(request.parameters))
 
                     array_mediator.validate_supported_space_efficiency(space_efficiency)
+                    if topologies:
+                        try:
+                            array_mediator.register_plugin('topology', '')
+                        except NotImplementedError:
+                            pass
```

Comment:
if its not an actual error can we not raise `NotImplementedError` from those mediators?